### PR TITLE
feat: Convert enriching-error-data to gatsby

### PR DESCRIPTION
**Pull request by dcramer**

### CODE REVIEW

File: gatsby/src/components/jsCdnTag.js [hunk 1]
```diff
@@ -1,19 +1,34 @@
 import React, { useState } from "react";
 
+let cachedVersionData = null;
```
Comment (dcramer) [inline gatsby/src/components/jsCdnTag.js:3]: realized it was doing a ton of requests - this stops that

File: gatsby/src/components/jsCdnTag.js [hunk 2]
```diff
@@ -1,19 +1,32 @@
...
+
 export default ({ apm = false }) => {
-  const [versionData, setVersionData] = useState(null);
+  const [versionData, setVersionData] = useState(cachedVersionData);
 
-  fetch(
-    "https://release-registry.services.sentry.io/sdks/sentry.javascript.browser/latest"
-  ).then(data => setVersionData(data));
+  if (!versionData) {
```
Comment (dcramer) [inline gatsby/src/components/jsCdnTag.js:8]: im confused as i thought i tested this but this clearly never worked

File: gatsby/src/includes/set-user/javascript.mdx [hunk 3]
```diff
@@ -0,0 +1,5 @@
+```javascript
```
Comment (PeloWriter) [inline gatsby/src/includes/set-user/javascript.mdx:1]: if this renders fine, no change. Or should it be "```js"?

Comment (dcramer) [reply gatsby/src/includes/set-user/javascript.mdx:1]: i believe behind the scenes its just an alias, so you can use either

**Pull request merged**